crates/red_knot/src/types.rs

```diff
@@ -389,6 +448,12 @@ impl ClassTypeId {
         }
     }
 
+    /// Get own class member or fallback to super-class member.
+    fn get_member(self, db: &dyn SemanticDb, name: &Name) -> QueryResult<Option<Type>> {
```

Comment:
Just realized we should actually call this `get_class_member` because ClassTypeId is used by both Class and Instance types, and the set of visible attributes can vary in each case.

(Not 100% sure this is the best way to handle it, but it's the naming convention currently used otherwise here, so let's stick to it until/unless we switch to a different approach.)